studio/components/grid/components/header/StatusLabel.tsx

```diff
@@ -31,8 +32,45 @@ const StatusLabel: React.FC<StatusLabelProps> = ({}) => {
 
   return (
     <div className="sb-grid-status-label">
-      {msg && <Typography.Text>{msg}</Typography.Text>}
-      {!msg && (
+      {status === 'saving' ? (
+        <Tooltip.Root delayDuration={0}>
+          <Tooltip.Trigger>
+            <IconLoader className="animate-spin mx-1" size={14} strokeWidth={2} />
```

Comment:
Not sure if just me but this isn't showing up.  Looks to need a text colour class. Shows up if I add one.

Reply:
fixed!